cognite/client/utils/_identifier.py

```diff
@@ -191,6 +192,11 @@ def unwrap_identifier(identifier: str | int | dict) -> str | int:
             return identifier["space"]
         raise ValueError(f"{identifier} does not contain 'id' or 'externalId' or 'space'")
 
+    @staticmethod
+    def extract_identifiers(dct: dict[str, Any]) -> dict[str, str | int]:
+        """An API payload might look like {"id": 1, "before": "2w-ago", ...}. This function extracts the identifiers"""
```

Comment:
Why is the `before: "2w-ago"` part of this example? Isn't this for identifiers?

Reply:
Because this function extracts identifiers only from whatever the API request (payload) may contain. So it was just the first thing to pop into my mind, could have been `limit=123` or whatever